File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/entities/adventurers/units/RoyalGuard.java
package it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.units;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.Skills;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.PotionDrinkerType;

/* JADX INFO: loaded from: classes3.dex */
public class RoyalGuard extends Adventurer {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer
    protected void configureStatistics() {
        this.maxLevel = 20;
        this.baseMaxHp = 130;
        this.baseConstitution = 20;
        this.baseIntelligence = 9;
        this.baseDexterity = 8;
        this.baseDefense = 20;
        this.baseMagicDefense = 20;
        this.threat = 3;
        this.counterattack = 0.3d;
        this.imageId = R.drawable.unit_royal_guard;
        this.idName = R.string.adventurer_royal_guard_name;
        this.idDescription = R.string.adventurer_royal_guard_description;
        this.passiveSkill = Skills.PASSIVE_SWORD_MASTERY_I;
        this.activeSkill = Skills.ACTIVE_MIGHTY_STRIKE;
        this.weaponType = R.string.type_sword;
        this.armorType = R.string.type_armor_heavy;
        this.potionDrinkerType = PotionDrinkerType.WARRIOR;
        this.nextClasses.add("RoyalSwordsman");
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/entities/adventurers/units/RoyalGuard.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.units

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.Skills
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.PotionDrinkerType

class RoyalGuard : Adventurer() {
    override fun configureStatistics() {
        maxLevel = 20
        baseMaxHp = 130
        baseConstitution = 20
        baseIntelligence = 9
        baseDexterity = 8
        baseDefense = 20
        baseMagicDefense = 20
        threat = 3
        counterattack = 0.3
        imageId = R.drawable.unit_royal_guard
        idName = R.string.adventurer_royal_guard_name
        idDescription = R.string.adventurer_royal_guard_description
        passiveSkill = Skills.PASSIVE_SWORD_MASTERY_I
        activeSkill = Skills.ACTIVE_MIGHTY_STRIKE
        weaponType = R.string.type_sword
        armorType = R.string.type_armor_heavy
        potionDrinkerType = PotionDrinkerType.WARRIOR
        nextClasses.add("RoyalSwordsman")
    }
}
